packages/playwright-test/types/test.d.ts

```diff
@@ -1055,6 +1055,10 @@ export interface FullConfig<TestArgs = {}, WorkerArgs = {}> {
    *
    */
   maxFailures: number;
+  /**
+   * Any JSON-serializable metadata that will be put directly to the test report.
+   */
+  metadata: any;
```

Comment:
Ah, so it is new, make it { [key: string]: string | number | boolean | { content: string|Buffer, contentType } } ?

Reply:
It's not new in TestConfig which is where it come from. We can change it there, too, though.